diff --git a/ctf/ctf.vcxproj b/ctf/ctf.vcxproj
index d4f17e3..03a6fe4 100644
--- a/ctf/ctf.vcxproj
+++ b/ctf/ctf.vcxproj
@@ -14,6 +14,7 @@
     <SccProjectName />
     <SccLocalPath />
     <ProjectGuid>{5B3D706A-4825-E865-5739-963E9E1758D0}</ProjectGuid>
+    <WindowsTargetPlatformVersion>10.0.19041.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
diff --git a/game/game.vcxproj b/game/game.vcxproj
index 4c9047c..2593dce 100644
--- a/game/game.vcxproj
+++ b/game/game.vcxproj
@@ -14,6 +14,7 @@
     <SccProjectName />
     <SccLocalPath />
     <ProjectGuid>{D1F2926B-5668-01CB-8504-34494E5D7D4B}</ProjectGuid>
+    <WindowsTargetPlatformVersion>10.0.19041.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
diff --git a/quake2.vcxproj b/quake2.vcxproj
index 2ba9693..88e9b3c 100644
--- a/quake2.vcxproj
+++ b/quake2.vcxproj
@@ -14,6 +14,7 @@
     <SccProjectName />
     <SccLocalPath />
     <ProjectGuid>{0093E157-9691-3A41-7274-BB2F4658BB05}</ProjectGuid>
+    <WindowsTargetPlatformVersion>10.0.19041.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
diff --git a/ref_gl/ref_gl.vcxproj b/ref_gl/ref_gl.vcxproj
index aee123c..aa567db 100644
--- a/ref_gl/ref_gl.vcxproj
+++ b/ref_gl/ref_gl.vcxproj
@@ -14,6 +14,7 @@
     <SccProjectName />
     <SccLocalPath />
     <ProjectGuid>{E076A42C-2F93-5D83-1082-EE2EC87BA60C}</ProjectGuid>
+    <WindowsTargetPlatformVersion>10.0.19041.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
diff --git a/ref_soft/ref_soft.vcxproj b/ref_soft/ref_soft.vcxproj
index 884236f..74a9b44 100644
--- a/ref_soft/ref_soft.vcxproj
+++ b/ref_soft/ref_soft.vcxproj
@@ -14,6 +14,7 @@
     <SccProjectName />
     <SccLocalPath />
     <ProjectGuid>{643FAAD2-53CC-08EF-95E6-45C082A7633C}</ProjectGuid>
+    <WindowsTargetPlatformVersion>10.0.19041.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
